definalizer/src/main/java/org/codice/junit/extension/builtin/DeFinalizer.java

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.junit.extension.builtin;
```

Comment:
built-in is the implementation side of things. You should never referenced it. The convention came from Spock when this was first written as part of Spock extensions.

Reply:
Actually, I think the convention was that the extensions included with Spock out-of-the-box are all located under `extension.builtin` packages (see http://spockframework.org/spock/docs/1.0/extensions.html: _In this chapter, we will first learn about Spock’s built-in extensions, and then dive into writing custom extensions._).
Should we consider moving those under `extension.custom` or something similar instead of using `builtin` since they are not builtin extensions?

Reply:
Please actually look at the `spock.lang.Ignore` extension which the documentation says it is a builtin extension. Its package is in spck.lang and its implementation is in `org.spockframework.runtime.extension.builtin.IgnoreExtension`.

Reply:
As for the fact that I moved this particular class from Spock to JUnit, I will rename the builtin package to impl and since `Definalizer` is actually reference from the API by design, I will move it to the junit package directly. For the Spock extensions, I will leave them as is.